dataprofiler/profilers/profile_builder.py

```diff
@@ -1374,10 +1393,25 @@ def _update_profile_from_chunk(self, data, sample_size,
         elif isinstance(data, list):
             data = pd.DataFrame(data)
 
-        if len(data.columns) != len(data.columns.unique()):
-            raise ValueError('`StructuredProfiler` does not currently support '
-                             'data which contains columns with duplicate '
-                             'names.')
+        duplicate_cols_given = len(data.columns) != len(data.columns.unique())
```

Comment:
this does not account for fuzzy lower, correct? We should maybe make a check as a separate function for the case of allowing user to set case sensitivity